integration-tests/main/src/test/java/io/quarkiverse/googlecloudservices/it/containers/StorageContainer.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkiverse.googlecloudservices.it.containers;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class StorageContainer implements QuarkusTestResourceLifecycleManager {
```

Comment:
Can you put this inside the test as it is done for firestore/bigtable ?
There is no real advantage of providing a test resource as it is not reused anywhere.

Reply:
I couldn’t find a way to update the properties this way. Any idea how we could dynamically update it within the test itself?

Reply:
Just use properties prefixed with `%test.` this is the easiest way